src/renderers/dom/fiber/ReactDOMFiber.js

```diff
@@ -545,6 +574,21 @@ var ReactDOM = {
   },
 };
 
+if (ReactDOMFeatureFlags.enableAsyncSubtreeAPI) {
+  ReactDOM.unstable_asyncRender = function(
+    element: ReactElement<any>,
+    container: DOMContainerElement,
+    callback: ?Function,
+  ) {
+    return render(element, container, true, callback);
+  };
+} else {
+  // We set this to null on the ReactDOM export, then delete if the feature
+  // flag is not enabled so that it's undiscoverable.
+  // TODO: Is there a better way to satisfy Flow?
+  delete ReactDOM.unstable_asyncRender;
```

Comment:
How do you expect to use the flag? I assumed that it would be used as a kill switch if we forgot to gate `unstable_asyncRender` somewhere, but if we invoked the kill switch, then that callsite (and others would just throw because we don't have the API.

Seems like this should just be a sync render by default, no?